video/out/vo_drm.c

```diff
@@ -43,6 +43,7 @@
     pixfmt2imgfmt(MP_SELECT_LE_BE(AV_PIX_FMT_X2RGB10LE, AV_PIX_FMT_X2RGB10BE))
 #define IMGFMT_XBGR2101010 \
     pixfmt2imgfmt(MP_SELECT_LE_BE(AV_PIX_FMT_X2BGR10LE, AV_PIX_FMT_X2BGR10BE))
+#define IMGFMT_YUYV pixfmt2imgfmt(AV_PIX_FMT_YUYV422)
```

Comment:
we only seem to define IMGFMT's for formats that are super common and/or we have a need to directly handle (hw mapping, osd, including them in `static` arrays).
since it's not necessary I wouldn't do it here.

Reply:
Ok, it is fine for me.